cylc/flow/scripts/lint.py

```diff
@@ -603,21 +603,21 @@ def list_wrapper(line: str, check: Callable) -> Optional[Dict[str, str]]:
             ' rather than job runner directive: ``{directive}``.'
         ),
         'rst': (
-            'Using ``[runtime][TASK]execution time limit`` is'
-            ' recommended in preference to using job runner'
-            ' directives because it allows Cylc to retain awareness'
-            ' of whether the job should have finished, even if contact'
-            ' with the target job runner\'s platform has been lost.'
-            ' \n\nThe following directives are considered equivelent to'
-            ' execution time limit:\n * '
+            "Using ``[runtime][TASK]execution time limit`` is"
+            " recommended in preference to using job runner"
+            " directives because it allows Cylc to retain awareness"
+            " of whether the job should have finished, even if contact"
+            " with the target job runner's platform has been lost."
+            " \n\nThe following directives are considered equivalent to"
+            " execution time limit:\n * "
```

Comment:
Might as well tighten this up - it's a bit verbose IMO.
```suggestion
            "Use ``[runtime][TASK]execution time limit`` to tell
            " Cylc when a job should finish. It is automatically"
            " translated to the right job runner directive:\n * "
```

Reply:
I've made it more concise but left in the explanation for why